### PR TITLE
BuildResidentialHPXML: HVAC input updates

### PR DESCRIPTION
**Pull request by shorowit**
## Pull Request Description

Some updates for HVAC systems:
1. Allows boilers as the secondary heating system (added test file)
2. Removes all detailed fan/pump power inputs; these are advanced features and we can just allow them to default.
3. Renames `heat_pump_mini_split_is_ducted` to `heat_pump_is_ducted`, for consistency with `cooling_system_is_ducted`.

Also clarifies a few misc argument values and units.

## Checklist

Not all may apply:

- [ ] EPvalidator.xml has been updated
- [ ] Tests (and test files) have been updated
- [ ] Documentation has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI

### CODE REVIEW

File: BuildResidentialHPXML/measure.rb [hunk 1]
```diff
@@ -1194,25 +1171,13 @@ def arguments(model)
...
-    arg.setUnits('W/ton')
-    args << arg
-
-    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heat_pump_fan_power_watts_per_cfm', false)
-    arg.setDisplayName('Heat Pump: Fan Power')
-    arg.setDescription('Blower fan power.')
-    arg.setUnits('W/CFM')
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('heat_pump_is_ducted', false)
+    arg.setDisplayName('Heat Pump: Is Ducted')
+    arg.setDescription("Whether the heat pump is ducted or not. Only used for #{HPXML::HVACTypeMiniSplitAirConditioner} and #{HPXML::HVACTypeEvaporativeCooler}.")
```
Comment (joseph-robertson) [inline BuildResidentialHPXML/measure.rb:1180]: Only used for HVACTypeHeatPumpMiniSplit?

**Review by joseph-robertson — CHANGES_REQUESTED** [review]

File: BuildResidentialHPXML/measure.rb (hunk 1 above)
Comment (shorowit) [reply BuildResidentialHPXML/measure.rb:1180]: Oops, nice catch. Bad copy paste.

File: BuildResidentialHPXML/measure.rb [hunk 2]
```diff
@@ -19,6 +19,7 @@
 require_relative '../HPXMLtoOpenStudio/resources/hvac'
 require_relative '../HPXMLtoOpenStudio/resources/lighting'
 require_relative '../HPXMLtoOpenStudio/resources/materials'
+require_relative '../HPXMLtoOpenStudio/resources/meta_measure'
```
Comment (joseph-robertson) [inline BuildResidentialHPXML/measure.rb:22]: @shorowit Why does this need to be added?

Comment (shorowit) [reply BuildResidentialHPXML/measure.rb:22]: Because you're now calling the `get_argument_values` method.

Comment (joseph-robertson) [reply BuildResidentialHPXML/measure.rb:22]: Do you know why it was working before?

Comment (shorowit) [reply BuildResidentialHPXML/measure.rb:22]: Not really. I got an error when trying to run one of the OSW's.

Comment (joseph-robertson) [reply BuildResidentialHPXML/measure.rb:22]: An existing osw that was running successfully on ci? Hmm

Comment (shorowit) [reply BuildResidentialHPXML/measure.rb:22]: I ran it manually via `openstudio run -w foo.osw` 🤷‍♂️

**Review by joseph-robertson — APPROVED** [review]

**Pull request merged**